pymatviz/utils.py

```diff
@@ -72,10 +72,10 @@ def pretty_label(key: str, backend: Backend) -> str:
     return symbol_mapping.get(key, {}).get(backend, key)
 
 
-def crystal_sys_from_spg_num(spg: int) -> CrystalSystem:
+def crystal_sys_from_spg_num(spg: float) -> CrystalSystem:
```

Comment:
Use `float` over `int` as we intend to accept either `integer or float with no decimal part`